Source/WebCore/platform/graphics/FontCascade.cpp

```diff
@@ -224,12 +224,7 @@ std::unique_ptr<DisplayList::DisplayList> FontCascade::displayListForTextRun(Gra
     ASSERT(!context.paintingDisabled());
     unsigned destination = to.value_or(run.length());
 
-    // FIXME: Use the fast code path once it handles partial runs with kerning and ligatures. See http://webkit.org/b/100050
-    CodePath codePathToUse = codePath(run);
-    if (codePathToUse != CodePath::Complex && (enableKerning() || requiresShaping()) && (from || destination != run.length()))
-        codePathToUse = CodePath::Complex;
```

Comment:
If I'm reading this correctly, this would only work if `USE(FREETYPE)` is false right? Otherwise wouldn't we be changing behavior if it's true since this logic would still be used? 

Do we need to change `FontCasecade::codePath` to account for that?